src/CMakeLists.txt

```diff
@@ -711,6 +711,8 @@ if(WITH_DPDK)
 endif(WITH_DPDK)
 
 add_library(common STATIC ${ceph_common_objs})
+set_target_properties(common PROPERTIES
+    COMPILE_FLAGS "-ftls-model=initial-exec")
```

Comment:
These lines appear unnecessary. According to `man gcc` the `initial-exec` should be the default TLS model when `-fPIC` is not specified:
>        -ftls-model=model
>
>           The default without `-fpic` is `initial-exec`; with `-fpic` the default
>           is `global-dynamic`.

Reply:
REMOVED.